src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java

```diff
@@ -71,6 +71,17 @@ public static String getStratification(Measure measure) {
     return null;
   }
 
+  public static String getStratificationDescription(Measure measure) {
+    if (CollectionUtils.isNotEmpty(measure.getGroups())) {
+      for (Group group : measure.getGroups()) {
+        if (CollectionUtils.isNotEmpty(group.getStratifications())) {
+          return group.getStratifications().get(0).getDescription();
+        }
```

Comment:
Why are we returning only the first description, Shouldn't this be a concatenation of all stratification descriptions ?

Reply:
Also Nice to have few unit test cases

Reply:
There's only going to be one description, covering all of the stratifications

Reply:
Given: A QDM Measure
When: on the Population Criteria tab, on a Criteria sub-tab, in the Stratifications section
Then: There is a Stratification Description field for every Stratification field (same layout as for QI-Core measures)

May be we need to get some clarification, The 1st AC reads different.

Reply:
Previous comment is wrong, fixed